testkit-backend/src/main/java/neo4j/org/testkit/backend/messages/requests/NewDriver.java

```diff
@@ -286,6 +305,15 @@ public static class NewDriverBody {
         private Long connectionAcquisitionTimeoutMs;
         private boolean encrypted;
         private List<String> trustedCertificates;
+        private BookmarkManagerConfig bookmarkManager;
+    }
+
+    @Setter
+    @Getter
+    public static class BookmarkManagerConfig {
+        private Map<String, Set<String>> initialBookmarks;
+        private boolean bookmarkSupplier;
+        private boolean notifyBookmarks;
```

Comment:
Testkit protocol will get updated in: https://github.com/neo4j-drivers/testkit/pull/506

This PR contains tests about the consumers and suppliers.

Reply:
Thank you, it will be updated separately.